### PR TITLE
Implement truncate-long-text for service class description and long description.

### PR DESCRIPTION
**Pull request by sg00dwin**
Updated views

Fixes https://github.com/openshift/origin-web-console/issues/2072

Related on https://github.com/openshift/origin-web-common/pull/214

### CODE REVIEW

**Comment by sg00dwin** [comment]
@spadgett rebased

File: app/views/overview/_service-instance-row.html [hunk 1]
```diff
@@ -135,11 +135,17 @@
...
+              <p class="service-description"><truncate-long-text
+                  limit="500"
+                  content="row.serviceClass.description"
+                  use-word-boundary="true"
+                  expandable="true"
+                  linkify="true">
+                </truncate-long-text></p>
+              <div ng-if="row.serviceClass.externalMetadata.documentationUrl || row.serviceClass.externalMetadata.supportUrl">
+                <a ng-if="row.serviceClass.externalMetadata.documentationUrl" ng-href="{{row.serviceClass.externalMetadata.documentationUrl}}" target="_blank" class="learn-more-link">View Documentation <i class="fa fa-external-link" aria-hidden="true"></i></a>
+                <a ng-if="row.serviceClass.externalMetadata.supportUrl" ng-href="{{row.serviceClass.externalMetadata.supportUrl}}" target="_blank" class="learn-more-link">Get Support <i class="fa fa-external-link" aria-hidden="true"></i></a>
```
Comment (spadgett) [inline app/views/overview/_service-instance-row.html]: Looks like a bad merge on your rebase. These should have `spec` in the path as above.

**Comment by sg00dwin** [comment]
@spadgett PTAL

File: app/views/browse/service-instance.html [hunk 2]
```diff
@@ -69,8 +69,20 @@ <h1 class="contains-actions">
...
+                            <p class="service-description"><truncate-long-text
+                                limit="500"
+                                content="serviceClass.description"
+                                use-word-boundary="true"
+                                expandable="true"
+                                linkify="true">
+                              </truncate-long-text></p>
+                            <p class="service-description"><truncate-long-text
+                                limit="500"
+                                content="serviceClass.externalMetadata.longDescription"
```
Comment (spadgett) [inline app/views/browse/service-instance.html]: It's now

```
serviceClass.spec.externalMetadata.longDescription
```

File: app/views/browse/service-instance.html [hunk 3]
```diff
@@ -69,8 +69,20 @@ <h1 class="contains-actions">
                           </dd>
                           <dt ng-if-start="serviceClass.spec.description || serviceClass.spec.externalMetadata.longDescription">Description:</dt>
                           <dd ng-if-end>
-                            <p class="pre-wrap" ng-bind-html="serviceClass.spec.description | linkify : '_blank'"></p>
-                            <p class="pre-wrap" ng-bind-html="serviceClass.spec.externalMetadata.longDescription | linkify : '_blank'"></p>
+                            <p class="service-description"><truncate-long-text
+                                limit="500"
+                                content="serviceClass.description"
```
Comment (spadgett) [inline app/views/browse/service-instance.html]: It's now

```
serviceClass.spec.description
```

**Comment by spadgett** [comment]
[merge]

**Pull request merged**